include/translated-object.php

```diff
@@ -387,11 +387,11 @@ public function current_user_can_synchronize( $id ) {
 		 *
 		 * @since 2.6
 		 *
-		 * @param $check Null to enable the capability check,
-		 *               true to always allow the synchronization,
-		 *               false to always disallow the synchronization.
-		 *               Defaults to true.
-		 * @param $id    The synchronization source object id.
+		 * @param bool|null $check Null to enable the capability check,
+		 *                         true to always allow the synchronization,
+		 *                         false to always disallow the synchronization.
+		 *                         Defaults to true.
```

Comment:
This would be clearer with class constants: `ALWAYS`, `NEVER`, `IF_USER_HAS_CAPABILITY`.